gae-interop-testing/README.md

```diff
@@ -24,11 +24,10 @@ Running the tests in GAE
 
 You can run the gradle task to execute the interop tests.
 ```bash
-# cd into either gae-jdk7 or gae-jdk8
+# cd into either gae-jdk8
```

Comment:
s/either//

Reply:
We have both kinds, western and country.